Include/cpython/pystate.h

```diff
@@ -364,3 +368,16 @@ typedef int (*crossinterpdatafunc)(PyObject *, _PyCrossInterpreterData *);
 
 PyAPI_FUNC(int) _PyCrossInterpreterData_RegisterClass(PyTypeObject *, crossinterpdatafunc);
 PyAPI_FUNC(crossinterpdatafunc) _PyCrossInterpreterData_Lookup(PyObject *);
+
+/* UNSTABLE API for stackful coroutines.
```

Comment:
What about hiding this behind an ifdef, it would technically be an api break, but would make the intention clearer: 

```c
#ifdef _PYTHON_C_UNSTABLE_FRAMESTACK_API
...
#endif
```